packages/api/lambdas/sf-event-sqs-to-db-records.js

```diff
@@ -135,67 +150,136 @@ const writeExecutionViaTransaction = async ({ cumulusMessage, trx }) =>
       arn: getMessageExecutionArn(cumulusMessage),
       cumulus_version: getMessageCumulusVersion(cumulusMessage),
       status: getMetaStatus(cumulusMessage),
-    });
+    })
+    .returning('cumulusId');
```

Comment:
What is `returning`? I couldn't find any meaningful search results when I looked

Reply:
This comes from the Knex query builder. It allows you to return a field when you are inserting a record:

http://knexjs.org/#Builder-returning